### ISSUE TITLE
False positive when using `Arg` from `mypy_extensions`

### ISSUE THREAD

**Issue by antifriz** [PyCQA/pycodestyle#1059]
The code below returns no errors when using `pylint` or `mypy` but `flake8==4.0.1` returns `F821 undefined name 'baz'`
 
```python
from collections.abc import Callable

from mypy_extensions import Arg


def foo(bar: Callable[[Arg(int, "baz")], None]) -> None:
    return bar(1)
```

**Comment by asottile** [comment]
`F` codes aren't provided by pycodestyle but by pyflakes

`Arg` isn't standardized (and is only kept in mypy-extensions for backward compatibility) and as far as I know is deprecated for `Protocol`s so it will not be implemented by pyflakes -- there's a few duplicates in the [pyflakes tracker](https://github.com/pycqa/pyflakes/issues) if you check there!
